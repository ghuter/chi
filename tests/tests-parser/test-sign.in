a :: fun ();
b :: fun (x: i32);
c :: fun (x: i32, y: i8);
d :: fun (x: i32, y: i32) i32;
e :: fun (x: i32, y: i32) ^i32;
f :: fun () i32;
g :: fun () ^i32;
