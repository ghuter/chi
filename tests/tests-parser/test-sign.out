SSIGN(a) : void
SSIGN(b, x : i32) : void
SSIGN(c, x : i32, y : i8) : void
SSIGN(d, x : i32, y : i32) : i32
SSIGN(e, x : i32, y : i32) : ^i32
SSIGN(f) : i32
SSIGN(g) : ^i32
